pkgs/games/dwarf-fortress/wrapper/default.nix

```diff
@@ -17,15 +17,15 @@ let
     else theme;
 
   # These are in inverse order for first packages to override the next ones.
-  pkgs = lib.optional (theme != null) ptheme
-         ++ lib.optional enableDFHack dfhack_
+  themePkg = lib.optional (theme != null) ptheme;
```

Comment:
Can you show what the error was? It may have been the compatible check? If so it may be we just need to add that attribute to the new themes.

Reply:
Oops that's not in the .json file but I think we can add it.

Reply:
Maybe it will be easier to just remove the compatibility check - it doesn't really make sense to have any way because we only support the latest version of dwarf fortress.

Reply:
Having mismatched versions of dfhack and df would make the game crash unpredictably, so I'm inclined to say yes. There's no reason *not* to update them in lockstep, of course; this just provides a sanity check to make sure we do so.

Reply:
I'm less sure about soundsense, but it does tend to update along with DF, so probably still yes. The reason I don't want to keep the test for the tilesets is that they don't update that predictably, *and* they by and large keep working anyway.